packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -7,7 +7,7 @@
                     <c-quantic-facet field="filetype" label="File Type" engine-id={engineId}></c-quantic-facet>
                     <c-quantic-numeric-facet field="ytlikecount" label="Youtube Likes" engine-id={engineId}></c-quantic-numeric-facet>
                     <c-quantic-date-facet field="date" label="Date" engine-id={engineId}></c-quantic-date-facet>
-                    <c-quantic-category-facet field="geographicalhierarchy" label="Country" engine-id={engineId}></c-quantic-category-facet>
+                    <c-quantic-category-facet field="atlgeographicalhierarchy" label="Country" engine-id={engineId}></c-quantic-category-facet>
```

Comment:
There are lots of changes related to the breadcrumb in this PR. I guess merging master to your branch would make the diff smaller.

Reply:
Yea I thought I did, bad merge I guess :( I'll fix it.

Reply:
All the breadcrumb stuff should be gone, my bad...